web/frontend/src/components/slam.vue

```diff
@@ -117,8 +133,13 @@ const fetchSLAMPose = (name: string): Promise<commonApi.Pose> => {
   });
 };
 
+const executeDeleteDestinationMarker = () => {
```

Comment:
why is this moving?

Reply:
a user is only allowed to press move if they've selected a destination
if a user has a selected destination and has pressed move **and** the request has completed in either success or failure we remove the original selected destination.